### PR TITLE
[PLAYER-5361] fix tests

### PR DESCRIPTION
**Pull request by MariskaS**
Changed arguments to ...args, because the library cannot compare the data of a non-real array of type argument with the data of the object type

### CODE REVIEW

**Comment by zelibobla** [comment]
Add the description of the PR, please. Explain briefly what was the issue and how your PR resolves it.

**Comment by ooyala-amohnatkin** [comment]
please also modify `test` command in package.json to enable tests on ci

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "common"
   ],
   "scripts": {
-    "test": "echo",
+    "test": "mocha test/unit-tests/*js --require test/unit-test-helpers/default_test_env.js --exit",
```
Comment (ooyala-amohnatkin) [inline package.json]: please revert to the previous state as here: https://github.com/ooyala/html5-common/pull/94/files

File: test/unit-test-helpers/default_test_env.js [hunk 2]
```diff
@@ -14,9 +14,15 @@ global.TestHelper = {
   }
 }
 
-global.jsdom = require("jsdom");
-global.document = jsdom.jsdom("<html><head></head><body>howdy</body></html>");
-global.window = document.createWindow();
+// global.jsdom = require("jsdom");
```
Comment (ooyala-amohnatkin) [inline test/unit-test-helpers/default_test_env.js]: remove commented code

**Pull request merged**